docs/asciidoc/advancedFeatures.adoc

```diff
@@ -833,6 +842,8 @@ using a `Context`.
 We first look back at our simple example from the introduction in a bit more detail, as
 the following example shows:
 
+//TODO change the examples
```

Comment:
just leaving a comment to make this TODO a part of the review :)

Reply:
done updated